modules/installation-supported-aws-machine-types.adoc

```diff
@@ -525,10 +559,15 @@ ifndef::aws-govcloud,aws-secret,aws-china,openshift-origin[]
 |x
 |x
 
+endif::aws-outposts[]
 |===
 ====
 endif::[]
 
+ifdef::aws-outposts[]
+Supported instance types in AWS outposts might be changed. For updated info you can check the link:https://aws.amazon.com/outposts/rack/features/#Compute_and_storage[Compute and Storage] page in AWS Outposts documents.
```

Comment:
Not sure I understood the comment. Do you mean that the admin may change the instance types or that AWS may change the list of supported instance types?
I think you mean the second one. If so please rephrase:
> AWS may change the list of supported instance types in AWS Outposts.

Reply:
ok